src/universal_dj_usb/generators/nml.py

```diff
@@ -242,40 +244,75 @@ def _add_track_to_collection(
 
         # Check if file exists (only add warning for debug purposes)
         if usb_path:
-            # Construct proper absolute path
             track_path_str = str(track.file_path)
             if track_path_str.startswith("/"):
-                track_path_str = track_path_str[1:]  # Remove leading slash
+                track_path_str = track_path_str[1:]
             absolute_file_path = usb_path / track_path_str
         else:
             absolute_file_path = track.file_path
 
         if not absolute_file_path.exists():
             warnings.append(f"File not found: {absolute_file_path}")
 
-        # Create entry element with title and artist as attributes
+        # Create entry attributes (title, artist, etc.)
         entry_attribs = {
             "MODIFIED_DATE": "2024/1/1",
             "MODIFIED_TIME": "0",
         }
-
-        # Add title and artist as attributes to ENTRY
         if track.title:
             entry_attribs["TITLE"] = track.title
         if track.artist:
             entry_attribs["ARTIST"] = track.artist
 
+        # Add Traktor-like attributes
+        entry_attribs["LOCK"] = "1"
+        entry_attribs["LOCK_MODIFICATION_TIME"] = datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
+        entry_attribs["AUDIO_ID"] = uuid.uuid4().hex[:32]
```

Comment:
no need for this attributes afaik.
can you explain what LOCK and LOCK_MODIFICATION_TIME mean? why would they be necessary in this scenario?

I'd rather have **minimal NML playlists** without _inventing_ anything. that's my testing conclusions, but I'd be happy to add as much metadata as we can as long as it's useful for live situations (mainly faster analysis).

eg.
`AUDIO_ID` it's not a random uuid but rather a representation of the audio itself (I think it's for waveform drawing). this would mean we should reverse engineer what Traktor generates by exporting playlists from Traktor and analyzing what it generates, and see if that's useful for faster analysis when loading tracks.